mcu_plugin.js

```diff
@@ -1896,6 +1896,10 @@ module.exports = function(RED) {
                             `${process.env["USERPROFILE"]}\\.espressif`,
                             `C:\\Espressif`
                         ]);
+                        env.LOCALAPPDATA =  ensure_env_path("LOCALAPPDATA", [
+                            `${process.env["LOCALAPPDATA"]};`,
```

Comment:
This line is redundant:
`ensure_env_path` already checks if `name` exists in `process.env[]`:
https://github.com/ralphwetzel/node-red-mcu-plugin/blob/3fd01e2acbbfe528b135fb99d02f5543ce398b2e/mcu_plugin.js#L1773-L1776